Source/Fuse.Nodes/Visual.BeginRemove.uno

```diff
@@ -84,7 +84,7 @@ namespace Fuse
 			}
 			else
 			{
-				args.Remove();				
+				UpdateManager.AddDeferredAction(args.Remove);
```

Comment:
Because I fixed a memory leak on 1.1 which requires new removal code in Each.
I think there might be an option though... Let me investigate.